cmd/minikube/cmd/start_flags.go

```diff
@@ -144,6 +145,7 @@ func initMinikubeFlags() {
 	startCmd.Flags().Bool(preload, true, "If set, download tarball of preloaded images if available to improve start time. Defaults to true.")
 	startCmd.Flags().Bool(deleteOnFailure, false, "If set, delete the current cluster if start fails and try again. Defaults to false.")
 	startCmd.Flags().Bool(forceSystemd, false, "If set, force the container runtime to use sytemd as cgroup manager. Currently available for docker and crio. Defaults to false.")
+	startCmd.Flags().String(startOutput, "", "Format to print stdout in. Options include: [json]")
```

Comment:
we should have a name for the default option . simmilar to other cli tools

Options include: text, json

and default is text

Reply:
for example az 

    --output -o         : Output format.  Allowed values: json, jsonc, none, table, tsv, yaml,
                          yamlc.  Default: json.

Reply:
done